exercises/practice/killer-sudoku-helper/.meta/test_template.ecr

```diff
@@ -0,0 +1,20 @@
+require "spec"
+require "../src/*"
+
+describe "<%-= to_capitalized(@json["exercise"].to_s) %>" do
+<%- @json["cases"].as_a.each do |cases| %>
+    <%- if cases["cases"]? -%>
+    <%- cases["cases"].as_a.each do |sub_case| %>
+    <%= status()%> "<%-= sub_case["description"] %>" do
+    killer_sudoku_helper = <%-= to_capitalized(sub_case["property"].to_s) %>.new(<%-= sub_case["input"]["cage"]["sum"] %>, <%-= sub_case["input"]["cage"]["size"] %>, <%-= sub_case["input"]["cage"]["exclude"] %> of Int32)
+    killer_sudoku_helper.solve.should eq <%-= sub_case["expected"] %>
+    end
+    <%- end -%>
+    <%- else -%>
+    <%= status()%> "<%-= cases["description"] %>" do
+    killer_sudoku_helper = <%-= to_capitalized(cases["property"].to_s) %>.new(<%-= cases["input"]["cage"]["sum"] %>, <%-= cases["input"]["cage"]["size"] %>, <%-= cases["input"]["cage"]["exclude"] %> of Int32)
+    killer_sudoku_helper.solve.should eq <%-= cases["expected"] %>
+    end
+    <%- end %>
+<% end %>
+end
```

Comment:
Could you indent this to improve readability? Since it's run through the crystal formatter after generation, the extra whitespace shouldn't matter, but it would make it easier for me to follow this.

Reply:
Some of them are written without being that formatted, but as you said the generator is setup to not care about how it is written. This is very nice since some languages generators are super sensative to this because they use intentation (python).

Although I think we should have some "standard" way of formatting these documents. If you are up to come up with a design so are you free to, otherwise I will do it at some point.

Also I changed my mind of hangman and replaced it with this one. Since hangman doesnt have conanical data.
Bank account will likely also be a no go.

If you want you want to do another exercise this month, you can do one of these once:

Affine Chiper
Bowling
Yatch
Linked list
Tournament
Two buckets
Book store

If you dont want so can I do it.

Reply:
I'll plan to grab one of those.